lib/samples/create_buffers_around_points/create_buffers_around_points.dart

```diff
@@ -0,0 +1,410 @@
+import 'package:arcgis_maps_sdk_flutter_samples/utils/sample_state_support.dart';
+import 'package:flutter/material.dart';
+
+class CreateBuffersAroundPoints extends StatefulWidget {
+  const CreateBuffersAroundPoints({super.key});
+
+  @override
+  State<CreateBuffersAroundPoints> createState() =>
+      _CreateBuffersAroundPointsState();
+}
+
+class _CreateBuffersAroundPointsState extends State<CreateBuffersAroundPoints>
+    with SampleStateSupport {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+
+  // A polygon that represents the valid area of use for the spatial reference.
+  late Polygon _boundaryPolygon;
+
+  // List of tap points.
+  final _bufferPoints = <PointAndRadius>[];
+
+  // Current status of the buffer.
+  var _status = Status.addPoints;
+
+  // Buffer radius.
+  var _bufferRadius = 100.0;
+
+  // Union status.
+  var _shouldUnion = false;
+
+  // A flag for when the settings bottom sheet is visible.
+  var _showSettings = false;
+
+  // Define the graphics overlays.
+  final bufferGraphicsOverlay = GraphicsOverlay();
+  final tapPointGraphicsOverlay = GraphicsOverlay();
+
+  // Fill symbols for buffer and tap points.
+  final bufferFillSymbol = SimpleFillSymbol();
+  final tapPointSymbol = SimpleMarkerSymbol();
+
+  final statePlanNorthCentralTexasSpatialReference =
+      SpatialReference(wkid: 32038);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: true,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                    onTap: onTap,
+                  ),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    ElevatedButton(
+                      onPressed: () {
+                        setState(() => _showSettings = !_showSettings);
+                      },
+                      child: const Text('Settings'),
+                    ),
+                    // A button to clear the buffers.
+                    ElevatedButton(
+                      onPressed: _bufferPoints.isEmpty
+                          ? null
+                          : () {
+                              clearBufferPoints();
+                              setState(() => _status = Status.addPoints);
+                            },
+                      child: const Text('Clear'),
+                    ),
+                  ],
+                ),
+                // Conditionally display the settings.
```

Comment:
I think this comment should be around line 88 where you define the settings button, and maybe make it "A button to conditionally display the settings."

Reply:
Moved it to line 88. Above the declaration of `ElevatedButton` for show Settings.